pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.black]
+line-length = 88
+target-version = ["py39",
+"py310",
+"py311",
+"py312",
+"py313"]
+
+[tool.isort]
+profile = "black"
+
+
+# 501 - line too long
+# E402 - module level import not at top of file
+# W503 - line break before binary operator
+# W293 - blank line contains whitespace
+# E302 - expected 2 blank lines, found N
+# W291 - trailing whitespace
+[tool.flake8]
+extend-ignore = [
+    "E501",
+    "E402",
+    "W503",
+    "W293",
+    "E302",
+    "W291"
```

Comment:
Any reason we picked these to ignore? 501 is fine. 402 I'm iffy on, but we can make it work. 503 is fine to skip. 293 i'd like to enforce. 302 i'd like to enforce. 291 i'd like to enforce.